include/solana.hpp

```diff
@@ -75,6 +75,19 @@ struct Version {
  */
 void from_json(const json &j, Version &version);
 
+struct EpochSchedule {
+  uint64_t firstNormalEpoch;
+  uint64_t firstNormalSlot;
+  uint64_t leaderScheduleSlotOffset;
+  uint64_t slotsPerEpoch;
+  bool warmup;
```

Comment:
Should implement 
```
    getEpoch(slot: number): number;
    getEpochAndSlotIndex(slot: number): [number, number];
    getFirstSlotInEpoch(epoch: number): number;
    getLastSlotInEpoch(epoch: number): number;
    getSlotsInEpoch(epoch: number): number;
```
as implemented in solana web3. They are essential to follow the schedule. Check solana repo for its implementation.

Reply:
@godmodegalactus should the tests be similar to[ this]( https://github.com/solana-labs/solana-web3.js/blob/53be88c38dd263df25eac37533b45f300c84e44c/test/epoch-schedule.test.ts#L21) ? Thanks